### ISSUE TITLE
Bug: Steps.Step not working

### ISSUE THREAD

**Issue by promise96319** [react-component/steps#369]
## Demo

```
<Steps current={1}>
  <Steps.Step title="first" />
  <Steps.Step title="second" />
  <Steps.Step title="third" />
</Steps>
```

##  Background

This demo in official docs is not working. The source code in `steps.tsx` not parse children props, but uses `items: StepsProps[]` to render children. `Steps.Step` doesn't work at all.

**Comment by julio-nt** [comment]
Yep, I've been encountering many problems with this stepper, it seems that the docs are outdated
